codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerOperationHandlerGenerator.kt

```diff
@@ -0,0 +1,222 @@
+                        async fn call(self, req: #{http}::Request<B>) -> #{http}::Response<#{SmithyHttpServer}::BoxBody> {
+                            let mut req = #{Axum}::extract::RequestParts::new(req);
+                            use #{Axum}::extract::FromRequest;
+                            use #{Axum}::response::IntoResponse;
+                            let input_wrapper = match $inputWrapperName::from_request(&mut req).await {
+                                Ok(v) => v,
+                                Err(r) => return r.into_response().map(#{SmithyHttpServer}::body::box_body)
+                            };
+                            $callImpl
+                            let output_wrapper: $outputWrapperName = output_inner.into();
+                            output_wrapper.into_response().map(#{SmithyHttpServer}::body::box_body)
+                        }
+                        """,
+                        *codegenScope
+                    )
+                }
+            }
+        }
+    }
+
+    /*
+     * Generates the trait bounds of the Handler trait implementation, depending on the presence of state
+     * and fallible vs infallible results.
+     */
+    private fun operationTraitBounds(operation: OperationShape, inputName: String, state: Boolean): String {
+        val inputFn = if (state) {
+            """S: Send + Clone + Sync + 'static,
+            Fun: FnOnce($inputName, $serverCrate::Extension<S>) -> Fut + Clone + Send + 'static,"""
+        } else {
+            "Fun: FnOnce($inputName) -> Fut + Clone + Send + 'static,"
+        }
+        val outputType = if (operation.errors.isNotEmpty()) {
+            "Result<${symbolProvider.toSymbol(operation.outputShape(model)).fullName}, ${operation.errorSymbol(symbolProvider).fullyQualifiedName()}>"
+        } else {
+            symbolProvider.toSymbol(operation.outputShape(model)).fullName
+        }
+        return """
+            $inputFn
+            Fut: std::future::Future<Output = $outputType> + Send,
+            B: $serverCrate::HttpBody + Send + 'static,
+            B::Data: Send,
+            B::Error: Into<$serverCrate::BoxError>,
+            $serverCrate::rejection::SmithyRejection: From<<B as $serverCrate::HttpBody>::Error>
+        """
+    }
+
+    /*
+     * This method is used to "generate" the OperationHandler struct, the Handler trait and all the
+     * code needed to support the indirection used to call the user defined functions, AKA operations
+     * implementations.
+     *
+     * TODO: remove this hacky function and move this piece of code into an inlinable crate. The crate should
+     *       be server specific, so we do not step on the client codegen feet. We choose to just add a static
+     *       string since there are changes needed in the Inlinable class to be able to support server specific
+     *       functionalities.
+     */
+    private fun renderStaticRust(writer: RustWriter) {
+        writer.rustTemplate(
```

Comment:
The "static" Rust has been removed from the runtime and put here to decouple the runtime with the operation handling and avoid a complex and messy implementation crossing the runtime and the codegen crates.